Chapters/6_Qualified_identifiers.md

```diff
@@ -18,13 +18,29 @@ The following *context qualifiers* are available:
 A "line" in the context of a file content refers to a sequence of characters that ends with a line break. This line can contain text, code, or any other form of data. In this specification, the line break is the ASCII LF character.
 The "lines" qualifier allows to designate a line range inside a content.
 The range can be a single line number, or a pair of line numbers separated by the ASCII `-` character.
+Line numbers start from 1.
```

Comment:
... "and range is inclusive, i.e. the fragment includes both the lines numbered as start and end of the range"

or something like this.